src/App.tsx

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string,
+};
+
+export class App extends Component<{}, State> {
+  state: State = {
+    pressedKey: '',
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.click);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.click);
+  }
+
+  click = (event: KeyboardEvent): void => {
```

Comment:
I think "click" is not good name for this
My advise use handler word like a part of name